micro_sam/util.py

```diff
@@ -463,7 +463,7 @@ def _compute_tiled_features_3d(predictor, input_, tile_shape, halo, f, pbar_init
 
 def _compute_2d(input_, predictor, f, save_path, pbar_init, pbar_update):
     # Check if the embeddings are already cached.
-    if save_path is not None and "input_size" in f.attrs:
+    if save_path is not None and "features" in f.attrs:
```

Comment:
`"features"` will never be in `f.attrs`, so this doesn't make sense.

Reply:
changed it back to "input_size" - there was a confusion on my side regarding the behavior of loading (tiled) embeddings: Loading tiled embeddings with no tiling information (given in the GUI) and vice versa leads to unclear messages. But this belongs to another issue: https://github.com/computational-cell-analytics/micro-sam/issues/332